attest/verifier/src/lib.rs

```diff
@@ -39,18 +39,24 @@ cfg_if::cfg_if! {
 
         /// Whether or not enclaves should be run and validated in debug mode
         pub const DEBUG_ENCLAVE: bool = true;
-        /// An array of zero-terminated signing certificate PEM files used as root anchors.
-        pub const IAS_SIGNING_ROOT_CERT_PEM: &str = IAS_SIM_ROOT_ANCHOR;
+        /// Root anchor PEM file for use with IAS
+        pub const IAS_SIGNING_ROOT_CERT_PEM: &str = SIM_ROOT_ANCHOR;
+        /// Root anchor PEM file for use with DCAP
+        pub const DCAP_ROOT_ANCHOR: &str = SIM_ROOT_ANCHOR;
     } else if #[cfg(feature = "ias-dev")] {
         /// Whether or not enclaves should be run and validated in debug mode
         pub const DEBUG_ENCLAVE: bool = true;
-        /// An array of zero-terminated signing certificate PEM files used as root anchors.
+        /// Root anchor PEM file for use with IAS
         pub const IAS_SIGNING_ROOT_CERT_PEM: &str = include_str!("../data/Dev_AttestationReportSigningCACert.pem");
+        /// Root anchor PEM file for use with DCAP
+        pub const DCAP_ROOT_ANCHOR: &str = include_str!("../data/DcapRootCACert.pem");
```

Comment:
There is no dev for DCAP, we will get the same signature for debug or sim, we need to check the debug bit when verifying.